lang/en/bigbluebuttonbn.php

```diff
@@ -429,6 +432,7 @@
 $string['view_section_title_presentation'] = 'Presentation file';
 $string['view_section_title_recordings'] = 'Recordings';
 $string['view_message_norecordings'] = 'There are no recording to show.';
+$string['view_message_oc_recordings'] = 'List of Recordings on Opencast';
```

Comment:
The heading for the recordings list on BBB is `Recordings`.
So, to align with this terminology, this should be:
```suggestion
$string['view_message_oc_recordings'] = 'Recordings on Opencast';
```